domains/test/service/src/domain.rs

```diff
@@ -456,6 +471,13 @@ where
             ReputationChange::new(i32::MAX, "Peer unbanned by test (2)"),
         );
     }
+
+    /// Take and stop the domain node and delete its database lock file
+    pub fn stop(self) -> Result<(), std::io::Error> {
+        // Remove the database lock file
+        std::fs::remove_file(self.base_path.path().join("paritydb/lock"))?;
```

Comment:
`/` is a Unix path separator, which only works on Linux and macOS. To generate a platform-independent path, use`join()` twice instead.